spec/system/admin/reports_spec.rb

```diff
@@ -1,4 +1,4 @@
-# frozen_string_literal: true
+    # frozen_string_literal: true
```

Comment:
What's this about?

Reply:
Oops. No idea how this got here... Fixing it in the next commit.